app/src/main/java/com/github/sdp/mediato/model/Location.java

```diff
@@ -2,6 +2,9 @@
 
 public class Location {
 
+    private double RADIUS = 100;

```

Comment:
Is this field here necessary as `LOCATION_RADIUS` is already defined in Utils.java?

Reply:
Nope, good catch !